#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From f80d5af3ad8705cf439db380cc846a647d67bcbd Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@intevation.de>
Date: Tue, 12 Aug 2014 21:22:23 +0200
Subject: [PATCH] Enable wildcard expansion with mingw-w64

    * g10/gpg.c: Define _dowildcard = -1;

--
    If this value is defined wildcard support in the c runtime
    is enabled again. See mingw-w64 crt/CRT_glob.c
---
 g10/gpg.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/g10/gpg.c b/g10/gpg.c
index 1a8e6e7..437e7f0 100644
--- a/g10/gpg.c
+++ b/g10/gpg.c
@@ -68,6 +68,9 @@
 #define MY_O_BINARY  0
 #endif

+#ifdef __MINGW32__
+int _dowildcard = -1;
+#endif

 enum cmd_and_opt_values
   {
--
1.9.1
